components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashReporterActivity.kt

```diff
@@ -72,6 +81,11 @@ class CrashReporterActivity : AppCompatActivity() {
     }
 
     private fun sendCrashReportIfNeeded(then: () -> Unit) {
+
+        /* Save the state of send checkbox into shared preferences.
+        */
```

Comment:
nit: I think we can get rid of this comment now. The following line is self explanatory :)